cirq-core/cirq/contrib/quimb/mps_simulator.py

```diff
@@ -576,6 +576,12 @@ def sample(
 class MPSState(ActOnArgs):
     """A state of the MPS simulation."""
 
+    @deprecated_parameter(
+        deadline='v0.16',
+        fix='Use classical_data and keyword args.',
```

Comment:
If there is a more explicit way to express this (something like "Replace `log_of_measurement_results=X` with `classical_data=X`") that is still concise, let's switch to that. Otherwise fine as-is.